File: DailyFlash Solutions/DailyFlash Solutions/40-DailyFlash_Solutions/28_Feb_Solution_Two/C++/prog5.cpp
#include<iostream>
#include<math.h>

int main() {
	float x1, x2, y1, y2;
	std::cout << "Enter x1 : ";
	std::cin >> x1;
	std::cout << "Enter y1 : ";
	std::cin >> y1;
	std::cout << "Enter x2 : ";
	std::cin >> x2;
	std::cout << "Enter y2 : ";
	std::cin >> y2;
	std::cout << "Dist : " <<sqrt((y2 - y1) * (y2 - y1) + (x2 - x1) * (x2 - x1));
}

File: DailyFlash Solutions/DailyFlash Solutions/46-DailyFlash_Solutions/13-Mar/Java/prog5.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		float x1, x2, y1, y2, x3, y3;
		System.out.println("Enter x1 : ");
		x1 = sc.nextFloat();
		System.out.println("Enter y1 : ");
		y1 = sc.nextFloat();
		System.out.println("Enter x2 : ");
		x2 = sc.nextFloat();
		System.out.println("Enter y2 : ");
		y2 = sc.nextFloat();
		System.out.println("Enter x3 : ");
		x3 = sc.nextFloat();
		System.out.println("Enter y3 : ");
		y3 = sc.nextFloat();
		System.out.println("SemiPerimeter = " + (Math.sqrt((y2 - y1) * (y2 - y1) + (x2 - x1) * (x2 - x1)) + Math.sqrt((y2 - y3) * (y2 - y3) + (x2 - x3) * (x2 - x3)) + Math.sqrt((y3 - y1) * (y3 - y1) + (x3 - x1) * (x3 - x1))) / 2);
	}
}
